scripts/bravo_csv.py

```diff
@@ -682,6 +683,13 @@ def main(lims, args):
 
 
 def calc_vol(art_tuple, logContext, checkTheLog):
+    art_workflows = []
+    for stage in art_tuple[0]['uri'].workflow_stages_and_statuses:
+        if stage[1] == 'IN_PROGRESS':
+            art_workflows.append(stage[0].workflow.name)
+    no_depletion_flag = False
+    if (art_tuple[0]['uri'].samples[0].project.udf.get('Library construction method') and 'no depletion' in art_tuple[0]['uri'].samples[0].project.udf['Library construction method']) or (art_tuple[0]['uri'].samples[0].project.udf.get('Library prep option') and 'No depletion' in art_tuple[0]['uri'].samples[0].project.udf['Library prep option']):
```

Comment:
Could have an extra variable  project which is`project = art_tuple[0]['uri'].samples[0].project`, so that this if is a bit easier to read. Not a necessity though.

Reply:
Fixed as suggested.